common/model/catalogue.ts

```diff
@@ -263,6 +263,22 @@ export type CatalogueAggregationBucket = {
   type: 'AggregationBucket';
 };
 
+export type CatalogueAggregationContributorsBucket = {
+  count: number;
+  data: {
+    agent: {
+      label: string;
+      type: string;
+    };
+    roles: string[];
```

Comment:
Let's ignore this for now as it's not actually a list of string, but a list of roles, but unnecessary.